solr/bin/solr

```diff
@@ -1314,6 +1314,7 @@ function start_solr() {
   SOLR_START_OPTS=('-server' "${JAVA_MEM_OPTS[@]}" "${GC_TUNE_ARR[@]}" "${GC_LOG_OPTS[@]}" "${IP_ACL_OPTS[@]}" \
     "${REMOTE_JMX_OPTS[@]}" "${CLOUD_MODE_OPTS[@]}" -Dsolr.log.dir="$SOLR_LOGS_DIR" \
     "-Djetty.port=$SOLR_PORT" "-DSTOP.PORT=$stop_port" "-DSTOP.KEY=$STOP_KEY" \
+    "-Djava.io.tmpdir=$(cd $TMPDIR; pwd -P)" \
```

Comment:
note this was enough to fix `./bin/solr start` but looks like SOLR_START_OPTS might not be used everywhere. for example in `stop_solr` or `run_tool` but we could apply a similar fix there.